fluss-rpc/src/main/java/com/alibaba/fluss/rpc/netty/client/NettyAuthenticationHandler.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2025 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.rpc.netty.client;
+
+import com.alibaba.fluss.exception.FlussRuntimeException;
+import com.alibaba.fluss.exception.RetriableAuthenticationException;
+import com.alibaba.fluss.rpc.messages.ApiMessage;
+import com.alibaba.fluss.rpc.messages.AuthenticateRequest;
+import com.alibaba.fluss.rpc.messages.AuthenticateResponse;
+import com.alibaba.fluss.security.auth.ClientAuthenticator;
+import com.alibaba.fluss.shaded.netty4.io.netty.channel.ChannelHandlerContext;
+import com.alibaba.fluss.shaded.netty4.io.netty.channel.ChannelInboundHandlerAdapter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Implementation of the channel handler to send authenticator. The client handler is not shared,
+ * there is a client handler instance for each channel connection.
+ */
+public class NettyAuthenticationHandler extends ChannelInboundHandlerAdapter {
+    private static final Logger LOG = LoggerFactory.getLogger(NettyAuthenticationHandler.class);
+    private static final Random RANDOM = new Random();
+    private static final long BASE_BACKOFF_MS = 500L;
+    private static final long MAX_BACKOFF_MS = 60000L;
```

Comment:
`BASE_BACKOFF_MS` -> `100L` (better initial latency) and `MAX_BACKOFF_MS` -> `5s` (not idle too long)